python-package/securexgboost/core.py

```diff
@@ -1623,6 +1623,16 @@ def predict(self, data, output_margin=False, ntree_limit=0, pred_leaf=False,
             pred_contribs), and the sum of the entire matrix equals the raw untransformed margin
             value of the prediction. Note the last row and column correspond to the bias term.
 
+        training : bool
+            Whether the prediction value is used for training.  This can effect
```

Comment:
What does this mean? Does this change affect us?

Reply:
It does in the sense that I had to modify all the APIs to incorporate it.
Essentially it seems to affect the behavior of training if the Booster is of type `dart` (it is `gbtree` by default).